drools-compiler/src/main/java/org/drools/compiler/builder/impl/KnowledgeBuilderImpl.java

```
@@ -936,6 +956,111 @@ private void addPackageFromKiePMML(PMMLCompiler compiler, Resource resource,
 		return null;
     }
 
+    private void addPMMLPojos(PMMLCompiler compiler, Resource resource) {
+        System.out.println("!!!!! PMMLCompiler version = "+compiler.getCompilerVersion()+" !!!!!");
+        KieFileSystem javaSource = KieServices.Factory.get().newKieFileSystem();
+        Map<String,String> javaSources = new HashMap<>();
+        Map<String,String> modelSources = null;
+        try {
+            modelSources = compiler.getJavaClasses(resource.getInputStream());
+        } catch (IOException e) {
+            // TODO Auto-generated catch block
```

This exception is swallowed here. Please handle it correctly.